### PR TITLE
feat (@core/home) : add home

### PR DESCRIPTION
**Pull request by marobakr**
1. add home

### CODE REVIEW

**Comment by vcpc** [comment]
1. Wrong padding-top at the HandsetPortrait screen
image
---
2. Collapsed images in the photo, make it just like the design
image
>  ### You can change the width and height for the dots and the logo according to the current screen, with the BreakpointObserver
---
3. The margin bottom in the HandsetPortrait screen
image
---
4. the margin right should be omitted in the HandsetPortrait screen
image
---
5. the padding left and right is 16px, and the logo is outside the container
image

File: src/app/core/components/home/home.component.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            <img class="w-100" src="./assets/Image-1.png" alt="logo" />
+            <img
+              [ngClass]="{ 'small-media': mobileMedia }"
+              class="home__dots position-absolute"
+              src="./assets/Dots.svg"
+            />
+            <img
+              [ngClass]="{ 'small-media': mobileMedia }"
+              class="home__outline position-absolute"
+              src="./assets/logo-outline.svg"
```
Comment (vcpc) [inline src/app/core/components/home/home.component.html:53]: this property string is more than 60 characters

File: src/app/core/components/home/home.component.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
+<main
+  [ngClass]="{
+    'small-media': mobileMedia
+  }"
+  class="home bg-main"
+>
+  <div class="container px-3">
+    <div
+      class="row gx-2 align-items-center justify-content-between home__row-mb"
```
Comment (vcpc) [inline src/app/core/components/home/home.component.html:9]: this property string is more than 60 characters

File: src/app/core/components/home/home.component.scss [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+    #{$refer}__dots {
+      /*
+ * Helper to change the home__images in small screens to be center
+  * 1. change box model
+ * 2. change Positioning
+*/
+      &.small-media {
+        /* Box Model */ /* 1 */
+        max-width: 56px;
```
Comment (vcpc) [inline src/app/core/components/home/home.component.scss:91]: Positioning is before Box Model

File: src/app/core/components/home/home.component.scss [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+ * Helper to change the home__images in small screens to be center
+ * 1. change Positioning
+*/
+      &.small-media {
+        /* Positioning */ /* 1 */
+        left: 0;
+      }
+    }
+
+    #{$refer}__dots {
```
Comment (vcpc) [inline src/app/core/components/home/home.component.scss:83]: Positioning is before Box Model

File: src/app/core/components/home/home.component.scss [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+      &.small-media {
+        margin    /* box model */ /* 1 */
+         : {
+          top: 24px;
+          bottom: 25px;
+        }
+      }
+    }
+  }
+
```
Comment (vcpc) [inline src/app/core/components/home/home.component.scss:47]: Positioning is before Box Model

File: src/app/core/components/home/home.component.ts [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+@Component({
+  selector: 'app-home',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './home.component.html',
+  styleUrl: './home.component.scss',
+})
+export class HomeComponent {
+  mobileMedia: boolean = false;
```
Comment (vcpc) [inline src/app/core/components/home/home.component.ts:13]: replace it with `HandsetPortrait`, that will be more meaningful

File: src/app/styles/components/_button.scss [hunk 7]
```diff
@@ -0,0 +1,19 @@
+button.btn-main-primary,
+.btn-main-primary:active {
```
Comment (vcpc) [inline src/app/styles/components/_button.scss:2]: `btn-main-primary:active` have no purpose with buttons

File: src/app/core/components/home/home.component.html (hunk 1 above)
Comment (vcpc) [inline src/app/core/components/home/home.component.html:53]: this property string is more than 60 characters

File: src/app/core/components/home/home.component.scss [hunk 8]
```diff
@@ -0,0 +1,182 @@
...
+    /*
+   * Helper to change the home in small screens
+ * 1. change box model
+  */
+    &.small-media {
+      /* Box Model */ /* 1 */
+      max-width: 313px;
+      height: 260px;
+
+      /* Position */
```
Comment (vcpc) [inline src/app/core/components/home/home.component.scss:58]: Positioning is before Box Model

File: src/app/core/components/home/home.component.scss [hunk 9]
```diff
@@ -0,0 +1,182 @@
...
+ * Helper to change the home__images in small screens to be center
+  * 1. change box model
+ * 2. change Positioning
+*/
+      &.small-media {
+        /* Box Model */ /* 1 */
+        max-width: 56px;
+        height: 56px;
+
+        /* Positioning */
```
Comment (vcpc) [inline src/app/core/components/home/home.component.scss:94]: Positioning is before Box Model

File: src/app/core/components/home/home.component.scss [hunk 10]
```diff
@@ -0,0 +1,182 @@
...
+        bottom: 38px; /* 2 */
+      }
+
+      /* Box Model */
+      max-width: 84px;
+      height: 84px;
+
+      /* Positioning */
```
Comment (vcpc) [inline src/app/core/components/home/home.component.scss:102]: Positioning is before Box Model

File: src/app/core/components/home/home.component.scss [hunk 11]
```diff
@@ -0,0 +1,182 @@
...
+ * Helper to change the home__images in small screens to be center
+  * 1. change box model
+ * 2. change Positioning
+*/
+      &.small-media {
+        /* Box Model */ /* 1 */
+        max-width: 104px;
+        height: 104px;
+
+        /* Positioning */ /* 2 */
```
Comment (vcpc) [inline src/app/core/components/home/home.component.scss:121]: Positioning is before Box Model

File: src/app/core/components/home/home.component.html [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+            />
+            <img
+              [ngClass]="{ 'small-media': mobileMedia }"
+              class="home__outline position-absolute"
+              src="./assets/logo-outline.svg"
+            />
+          </figure>
+          <div
+            [ngClass]="{ 'small-media': mobileMedia }"
+            class="home__frame p-2 d-flex align-items-center text-main-primary gap-2 color-main border-secondary"
```
Comment (vcpc) [inline src/app/core/components/home/home.component.html:58]: this property string is more than 60 characters

**Comment by vcpc** [comment]
1. because you replaced the `.text-main` rule with the `.text-main-primary` rule, the color is removed from the nav links, just go to the nav links and change the class from `text-main` to `text-main-primary`, since the second name is more semantic
---
2. bad comment writing:
image

**Comment by marobakr** [comment]
1. home
1.1 handle Word Wrap with 60 characters
1.2.replace mobile media to HandsetPortrait
1.3.change order position and box model at the line (85,94,102,121) @ / home.scss
1.4.handle comment

2.navbar
2.1. change class text-main to text-amin-primary in nav

File: src/app/core/components/home/home.component.html [hunk 13]
```diff
@@ -0,0 +1,70 @@
+<main
+  [ngClass]="{
+    'small-media': handsetPortrait
+  }"
+  class="home bg-main"
+>
+  <div class="container px-3">
+    <div
+      class="row gx-2 align-items-center justify-content-between home__row-mb"
```
Comment (vcpc) [inline src/app/core/components/home/home.component.html:9]: this property string is more than 60 characters

File: src/app/core/components/home/home.component.html [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+            />
+            <img
+              [ngClass]="{ 'small-media': handsetPortrait }"
+              class="home__outline position-absolute"
+              src="./assets/logo-outline.svg"
+            />
+          </figure>
+          <div
+            [ngClass]="{ 'small-media': handsetPortrait }"
+            class="home__frame p-2 d-flex align-items-center text-main-primary gap-2 color-main border-secondary"
```
Comment (vcpc) [inline src/app/core/components/home/home.component.html:58]: this property string is more than 60 characters

File: src/app/core/components/home/home.component.scss [hunk 15]
```diff
@@ -0,0 +1,181 @@
...
+        right: 0;
+        top: 32px;
+        bottom: 24px;
+      }
+      /**
+        * Helper to change the home in small screens
+        * 1. change box model
+        */
+      &.small-media {
+        margin    /* box model */ /* 1 */
```
Comment (vcpc) [inline src/app/core/components/home/home.component.scss:39]: invalid code writing

**Comment by vcpc** [comment]
1. broken navbar:
image
---
2. you reformatted the code of the `navbar.component.html` file

**Comment by marobakr** [comment]
feat(@core/home): home

1.home
1.1 handle Word Wrap with 60 characters
1.2.handle comment

2.navbar
2.1. reformatted navbar

**Pull request merged**